lighthouse-core/lib/i18n/i18n.js

```diff
@@ -183,6 +183,8 @@ const _ICUMsgNotFoundMsg = 'ICU message not found in destination locale';
  */
 function _formatIcuMessage(locale, icuMessageId, fallbackMessage, values) {
   const localeMessages = LOCALES[locale];
+  if (!localeMessages) throw new Error(`Unsupported locale '${locale}'`);
```

Comment:
this isn't something that should ever happen outside of tests since in a normal run we specifically pick which locale to use from the keys of `LOCALES` itself.

...but if it does somehow happen in a normal run, this is a strictly better error message than `"Cannot read property 'lighthouse-core/lib/lh-error.js | pageLoadFailed' of undefined"` anyways, so seems ok.